pkg/ibmvpc/examples/out/explain_out/tgwAnotherExampleEnabledConn_all_vpcs_explain.txt

```diff
@@ -1,7 +1,7 @@
 Explaining connectivity from ky-vsi0-subnet5 to ky-vsi0-subnet11
 ================================================================
 
-Allowed connections from ky-vsi0-subnet5[10.240.9.4] to ky-vsi0-subnet11[10.240.80.4]: All Connections
+Allowed connections from test-vpc0-ky/ky-vsi0-subnet5[10.240.9.4] to test-vpc1-ky/ky-vsi0-subnet11[10.240.80.4]: All Connections
 
 Path:
 	ky-vsi0-subnet5[10.240.9.4] -> security group sg1-ky -> subnet5 -> network ACL acl3-ky -> 
```

Comment:
VSI name in the path should not be using `ExtendedName` format?

Reply:
No, in the path I believe they should not be using the `ExtendedName`  format. The VPC is part of the path anyways. I find it clearer this way.